R/asnumeric.R

```diff
@@ -172,30 +172,23 @@ asNumericVector <- function(t)
 {
     v <- gsub(",", "", TrimWhitespace(as.vector(t)))
     v <- gsub("^\\$", "", v)
-    result <- suppressWarnings(as.numeric(v))
+
+    # Convert parentheses to negative numbers
+    patt <- "^\\(\\$?[0-9.]+%?)$"
+    is.neg <- regexpr(patt, v) > 0
```

Comment:
Could use `grepl`?

Reply:
Yep, I've replaced it with `grepl` in eb80adb